cmn/daq_common.h

```diff
@@ -11,6 +11,7 @@ typedef int32_t daq_Strain_t;
 typedef uint32_t daq_BatteryV_t;
 typedef int32_t daq_ThrottlePos_t;
 typedef int32_t daq_FuelPressure_t;
+typedef int32_t daq_GPSVehicleSpeed_t;
```

Comment:
looking at the library[ source code](https://github.com/adafruit/Adafruit_GPS/tree/master/src) it seems that speed is defined as `float` so as Teensy is 32 bit, sets define `daq_GPSVehicleSpeed_t` as float32_t